inc/saitypes.h

```diff
@@ -557,6 +557,49 @@ typedef struct _sai_attribute_t {
 } sai_attribute_t;
 
 /**
+ * @brief Bulk objects creation.
+ *
+ * @param[in] switch_id SAI Switch object id
+ * @param[in] object_count Number of objects to create
+ * @param[out] object_id List of object ids returned
+ * @param[in] attr_count List of attr_count. Caller passes the number
+ *         of attribute for each object to create.
+ *
+ * @param[in] attrs List of attributes for every object.
+ * @param[out] object_statuses List of status for every object. Caller needs to allocate the buffer.
+ *
+ * @return #SAI_STATUS_SUCCESS on success when all objects are created or #SAI_STATUS_FAILURE when
+ * any of the objects fails to create. When there is failure, Caller is expected to go through the
+ * list of returned statuses to find out which fails and which succeeds.
+ */
+
+typedef sai_status_t (*sai_bulk_object_create_fn)(
+        _In_ sai_object_id_t switch_id,
+        _In_ uint32_t object_count,
+        _Out_ sai_object_id_t *object_id,
```

Comment:
Would it look cleaner to have the two _Out_'s in the end of the param list. 
_Out_ sai_object_id_t *object_id, _Out_ sai_status_t *object_statuses ?

Everything else looks fine.